tests/installation/user_settings.pm

```diff
@@ -44,7 +44,7 @@ sub run {
         record_soft_failure('boo#1122804 - Typing issue with fullname') unless match_has_tag('inst-userinfostyped-expected-typefaces');
         $retry++;
     } while (($retry < $max_tries) && !match_has_tag('inst-userinfostyped-expected-typefaces'));
-    assert_screen('inst-userinfostyped-expected-typefaces');    # fail if mistyped
+    assert_screen('inst-userinfostyped-expected-typefaces') if (check_var('SLE_PRODUCT', 'sles') && is_sle('15-SP2+'));    # fail if mistyped
```

Comment:
I believe that this is relevant check and without it investigation will be more complicated. Also switching to the user console might be affected. So I would propose to at least record info. Maybe soft-failures are enough.
Not sure why we want to limit scope to SLES only. So let's ask someone outside of QSF, as for us I would keep it as it is and you don't change it for our scope, so I have nothing against this change.

Reply:
Because of this conversation: https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/9780#issuecomment-599897936

Reply:
one of those test is about SLES, so I think this check may not help. 
I have a idea, how about we don't use the famous name BWM, we use bwm in stand for those task only care about qcow2 and don't care about check point during installation?

Reply:
We could have setting that changes de full name to be used. So, teams that want to ignore the typing issues for fullname can set a different fullname.

Reply:
> We could have setting that changes de full name to be used. So, teams that want to ignore the typing issues for fullname can set a different fullname.

yeah, I like this solution if it could be implemented.

Reply:
@SeroSun, please, see https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/9808